index.js

```diff
@@ -4,16 +4,16 @@ const discovery = require('hyperdiscovery')
 
 module.exports = function (archiveOrKey, opts) {
   var archive = archiveOrKey.key ? archiveOrKey : createArchive(archiveOrKey)
+  var feed = (archive.content) ? archive.content : archive
```

Comment:
iirc `hyperhealth` does nothing in that case - e.g. `archive.content` is needed for https://github.com/karissa/hyperhealth/blob/master/index.js#L36-L37 - what should the value of `.content` be?